src/types/common.ts

```diff
@@ -15,6 +15,12 @@ export {
   ZeroExSignedFillOrder,
 } from 'set-protocol-utils';
 
+export interface AddressTokenFlows {
```

Comment:
Funny name. In the contract, I think TokenFlow by itself defined the 3 items. Also instead of `combinedTokenAddresses`, I just had `tokens` or `addresses`